lib/table_sync/event_actions.rb

```diff
@@ -33,7 +33,8 @@ def update(data) # rubocop:disable Metrics/MethodLength
 
     def destroy(data)
       attributes = data.first || {}
-      target_attributes = attributes.select { |key| target_keys.include?(key) }
+      prevent_inclomplete_destroy_event!(attributes)
```

Comment:
I think it's better to just pass `target_attributes` to `prevent_inclomplete_destroy_event!` and to check if `target_attributes.keys == target_keys` there. Same

Reply:
yep